docs/products/clickhouse/howto/data-service-integration.md

```diff
@@ -1,134 +1,215 @@
+
+1. Select **Service types** of Aiven-managed services to integrate with. See
+   [supported data source types](/docs/products/clickhouse/concepts/data-integration-overview#supported-data-source-types)
+   for supported service types.
+1. Select **Data service** of the chosen type from the list of services available for
+   integration, and click **Continue**.
+1. Click either **Enable without databases** or **Add databases**.
+
+   :::note
+   If you prefer to create a data service integration without adding integration databases,
+   you can create integration databases for your service any time later. See
+   [Manage Aiven for ClickHouse® integration databases](/docs/products/clickhouse/howto/integration-databases)
+   for guidance on how to do that.
+   ::::
+
+   - Enable your integration with databases:
+
+     Click **Add databases**, enter database and schema names, and click **Enable**.
+
+     You can preview the created databases by selecting <ConsoleLabel name="databasestables"/>
+     from the sidebar.
+
+   - Enable your integration without databases:
+
+     Click **Enable without databases**.
+
+     You can preview the created integration by selecting <ConsoleLabel name="overview"/>
+     from the sidebar.
+</TabItem>
+</Tabs>
+
+### Integrate with external data sources {#integrate-external-services}
+
+For integration with external data sources, Aiven for ClickHouse offers two methods:
+
+- [Managed databases](/docs/products/clickhouse/concepts/data-integration-overview.md#managed-databases-integration)
+- [Managed credentials](/docs/products/clickhouse/concepts/data-integration-overview.md#managed-credentials-integration)
+
+To create an integration with an external data source in the **Data service integrations**
+wizard:
+
+1. Select **Service types** of an external data source to integrate with. See
+   [supported data source types](/docs/products/clickhouse/concepts/data-integration-overview#supported-data-source-types)
+   for supported service types.
+1. Select an external **Data service** of the chosen type from the list of services available
+   for integration, and click **Continue**.
+1. Select an integration method, either **Managed databases**  or **Managed credentials**,
+   and click **Continue**.
+1. Integrate with managed databases or with managed credentials:
+
+<Tabs groupId="group2">
+<TabItem value="1" label="Integrate with managed databases" default>
+The managed databases integration uses databases engines and, when enabled,
+automatically creates databases in your Aiven for ClickHouse, where you can ingest your
+external data.
+
+1. Select **Managed databases**, and click **Continue**.
+1. Click **Add databases** to have custom databases created along with the integration, or
+   click **Enable without databases** to integrate with no custom databases created.
+
+   :::tip
+   If you choose to have databases created automatically, you can populate them with your
```

Comment:
The databases that are created automatically don't need to be populated I think. When you execute a ``SELECT`` statement, you fetch the data from the remote database server. So it would be more appropriate to say here "you can query them using a statement similar to the following", or something to that extent.